packages/core/src/state-management/repository.ts

```diff
@@ -363,7 +363,9 @@ export class Repository {
     const STREAM_CONTENT = {
       model: state$.value.metadata.model,
       streamID: state$.id,
-      controller: state$.value.metadata.controller,
+      controller: state$.value.metadata.controllers[0],
```

Comment:
Confirmed that this is right and makes sense.  `state$` is a [StreamState](https://github.com/ceramicnetwork/js-ceramic/blob/5fc4e6157f52ef513abda2845005673f1955ae48/packages/common/src/stream.ts#L137), `metadata` is a [StreamMetadata](https://github.com/ceramicnetwork/js-ceramic/blob/5fc4e6157f52ef513abda2845005673f1955ae48/packages/common/src/stream.ts#L90) and it has a `controllers` array